translations/localify/ui.json

```diff
@@ -1997,5 +1998,158 @@
+    "出走条件達成！": "Entry req met!",
+    "特にありません": "No special conditions",
+    "おまかせ選択しました": "Auto-selected",
+    "+{0} 人": "+{0} Fans",
+    "ログがありません": "No logs",
+    "レンタルを含める": "Include rentals",
+    "{0}〜{1}人": "{0}〜{1} Fans",
+    "このウマ娘で１度も育成を完了していません": "This Uma Musume has not completed any trainings",
+    "このシナリオでの育成を行っていません": "No trainings for this scenario",
+    "ストーリーマーク設定": "Bookmark Story",
+    "変更した内容でストーリーマークを設定します\nよろしいですか？": "Changing the specified story bookmarks. \nContinue?",
+    "マークするストーリーを選択してください": "Please select stories to bookmark",
+    "マーク設定": "Set Bookmarks",
+    "ねぇ～！ちょー強いカブトムシ、\n捕まえに行こうよ～！\nゴルシも持ってるんだって～！": "Hey～! Let's go catch a strong beetle～! \nGolshi has one too～!",
+    "ねぇ～！ちょー強いカブトムシ、捕まえに行こうよ～！　ゴルシも持ってるんだって～！": "Hey～! Let's go catch a strong beetle～! \nGolshi has one too～!",
+    "ちゃんと見ててね、トレーナー！\nぱぱーっと勝ってくるから、\n瞬きしたら見逃しちゃうかもよ～？": "Watch carefully, trainer! \nI'll win in a flash; if you blink, \nyou might miss it～",
+    "ちゃんと見ててね、トレーナー！　ぱぱーっと勝ってくるから、瞬きしたら見逃しちゃうかもよ～？": "Watch carefully, trainer! \nI'll win in a flash; if you blink, \nyou might miss it～",
+    "予 想": "Predict",
+    "トレーナー、お疲れ様っ。\nうむうむ、今日もボクのために\nご苦労であ～る♪": "Trainer, good job! \nHmmm, you worked hard for me \nagain today～♪",
+    "トレーナー、お疲れ様っ。うむうむ、今日もボクのためにご苦労であ～る♪": "Trainer, good job! \nHmmm, you worked hard for me \nagain today～♪",
+    "どうせボクが勝っちゃうし、\n今のうちに表彰台での台詞\n考えとこっかな～♪": "I'm going to win anyway, \nso maybe I'll think of a line to \nsay on the podium right now～♪",
+    "どうせボクが勝っちゃうし、今のうちに表彰台での台詞考えとこっかな～♪": "I'm going to win anyway, \nso maybe I'll think of a line to \nsay on the podium right now～♪",
+    "トレーニングの効果が20％上昇するレース中、基礎能力が4％上昇する": "20% increase in training effectiveness\n4% increase in basic ability during a race",
+    "トレーニングの効果が10％上昇するレース中、基礎能力が2％上昇する": "10% increase in training effectiveness\n2% increase in basic ability during a race",
+    "トレーニングの効果に増減なしレース中、基礎能力の増減なし": "No change in training effectiveness\nNo change in basic ability during a race",
+    "トレーニングの効果が10％減少するレース中、基礎能力が2％減少する": "10% reduction in training effectiveness\n2% reduction in basic ability during a race",
+    "トレーニングの効果が20％減少するレース中、基礎能力が4％減少する": "20% reduction in training effectiveness\n4% reduction in basic ability during a race",
+    "よ～し、絶好調！\nふふんっ、テイオー様の最強伝説は\n今回も破られそうにないねっ！": "Al～right, I'm in top condition! \nHehe, Teio-sama's greatest legend \nisn't about to be broken this time!",
+    "よ～し、絶好調！　ふふんっ、テイオー様の最強伝説は今回も破られそうにないねっ！": "Al～right, I'm in top condition! \nHehe, Teio-sama's greatest legend \nisn't about to be broken this time!",
+    "←現在": "Current",
+    "Classic 10月  (Late)": "Classic Oct  (Late)",
+    "Classic 5月  (Late)": "Classic May  (Late)",
+    "Classic 4月  (Early)": "Classic Apr  (Early)",
+    "Classic 1月  (Late)": "Classic Jan  (Late)",
+    "Senior 4月  (Late)": "Senior Apr  (Late)",
+    "Senior 11月  (Late)": "Senior Nov  (Late)",
+    "Senior 12月  (Late)": "Senior Dec  (Late)",
```

Comment:
These look like dynamic combinations and can be removed. The individual parts are already translated.